rtengine/dcp.cc

```diff
@@ -393,6 +393,13 @@ double xyCoordToTemperature(const std::array<double, 2>& white_xy)
     return res;
 }
 
+struct cJSON_deleter {
+    template<typename T>
+    void operator()(T *t) {
+        cJSON_Delete(const_cast<typename std::remove_const<T>::type*>(t));
```

Comment:
Sure the template is needed? `cJSON_Delete()` only takes a `cJSON*`.

See comment below.

Reply:
True, this version lets you pass a const pointer (unless I goofed it up).  Future proofing this if it gets reused.  Although, if you want the lambda, this is unneeded anyway.

Reply:
True, this lets you pass a `const` pointer I think.  Tried to future-proof this a little.  If you want the lambda, it'll all go away.